ecodan-ha-local/ehal_http.cpp

```diff
@@ -173,6 +174,11 @@ namespace ehal::http
         else
             page.replace(F("{{cool_enabled}}"), "");
 
+        if (config.UniqueId)
```

Comment:
I guess because this is a string not a boolean it might be clearer if we wrote it as:
```
if (config.UniqueId.length() > 0)
```